libs/carrier-portal/ui/src/lib/pipes/phone-number.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'phoneNumber',
+  standalone: true,
+})
+export class PhoneNumberPipe implements PipeTransform {
+  transform(phoneNumber: string | undefined | null): string {
+    return phoneNumber ? this.formatPhoneNumber(phoneNumber) : 'No Number';
+  }
+
+  private formatPhoneNumber(number: string) {
+    if (number.length === 7) {
+      const top = number.slice(0, 2);
+      const rest = number.slice(3, 6);
+      return `${top}-${rest}`;
+    }
+    if (number.length === 10) {
+      const ac = number.slice(0, 2);
+      const top = number.slice(3, 5);
+      const rest = number.slice(6, 9);
+      return `(${ac}) ${top}-${rest}`;
+    }
```

Comment:
Is it worth to also format phone number when it has country code in it?

Reply:
Hm...I don't know if we include those let me research.

Reply:
I've added handling for those with country codes.